src/main/features/claimant-response/views/confirmation.njk

```diff
@@ -78,13 +78,15 @@
       {% elseif repaymentPlanOrigin === 'COURT' %}
         {% include "./confirmation/court-suggested-payment-plan.njk" %}
       {% elseif claim.claimantResponse.type === ClaimantResponseType.ACCEPTATION %}
-        {% if claim.response.responseType === ResponseType.PART_ADMISSION.value and claim.response.paymentIntention.paymentOption === PaymentOption.IMMEDIATELY %}
+        {% if claim.response.responseType === domain.ResponseType.FULL_DEFENCE %}
+          <p>{{ t('The claim is now ended. We’ve emailed {{ defendantName }} to tell them.', { defendantName: claim.claimData.defendant.name }) }}</p>
+        {% elseif claim.response.responseType === ResponseType.PART_ADMISSION.value and claim.response.paymentIntention.paymentOption === PaymentOption.IMMEDIATELY %}
```

Comment:
can you please swap the `ResponseType.PART_ADMISSION.value` to use `domain.ResponseType.PART_ADMISSION` to make it consistent?

Reply:
ok,  i have done that for you :) but as you know that line of code is not related to my story.

Reply:
thanks!